### ISSUE TITLE
DllNotFoundException: hidapi assembly <Unknown assembly and unknown type error>

### ISSUE THREAD

**Issue by samster1990** [Looking-Glass/JoyconLib#53]
Hi I have been trying to connect my switch controllers using Looking-Glass/JoyconLib package. But I am getting this error. Can anyone suggest a fix for this issue. I am not very familiar to Unity. 
Thanks in advance.
<img width="832" alt="Screenshot 2023-01-16 at 19 52 48" src="https://user-images.githubusercontent.com/25251444/212756927-4c49d499-b3b8-4771-8e1c-34171a2a18a1.png">

**Comment by Olyxr** [comment]
Hey. Did you figure out how to fix the issue? I'm having the same issue (On Mac)

**Comment by samster1990** [comment]
> 

No I didn't find any solution so far. I tried checking on different unity versions but it doesn't work. I will try again to troubleshoot but let me know if you find any fix.

**Comment by Olyxr** [comment]
Yeah. I also tried that. It might have something to do with the latest version of MacOS. I think Apple changed the way game controllers are handled. That is if you also use Mac, and you also are on MacOS Ventura.

**Comment by samster1990** [comment]
> 

Yes I am also using mac. It might be Mac issue. By the way did your switch controllers connect to your mac? My controllers are connecting to properly but not within unity.

**Comment by Olyxr** [comment]
Right. Yes, my Joycons connect and work just fine. I often use them for emulators. The problem for me is also only happening within Unity.
